django_reference_app/django_reference_app/templates/infopage.html

```diff
@@ -77,18 +79,20 @@ <h2 style="font-size:24px;font-weight:400;line-height:1.25;color: #6f777b">Congr
       <br>
       <h2 style="font-size:24px;font-weight:600;line-height:1.25;">Getting started with the Cloud Platform</h2> 
       <br>
-      <p>This app will give information on the new cloud platform solution</p>
+      <h3>Hello {{ name }}</h3>
+      <p>This site will give information on the new cloud platform solution</p>
       <p>These are the follwoing repos and resources you will need to use the new platform</p>
       <ul style="list-style:circle;padding-left:10px">
         <li><a href="https://kubernetes.io/">Kubernetes documentation</a></li>
-        <li><a href="https://github.com/ministryofjustice/k8s-nonprod-environments">Non-production cluster repo</a></li>
+        <li><a href="git@github.com:ministryofjustice/k8s-nonprod-environments.git">Non-production cluster repo</a></li>
```

Comment:
This link is not going to work -- it is a git link for cloning, you'll need the http link: https://github.com/ministryofjustice/k8s-nonprod-environments/